cms/djangoapps/contentstore/outlines.py

```diff
@@ -79,6 +79,27 @@ def _error_for_not_sequence(section, not_sequence):
     )
 
 
+def _error_for_duplicate_child(section, duplicate_child):
+    """
+    ContentErrorData when we run into a child of Section that's defined in a
+    previous section.
+
+    Has to be phrased in a way that makes sense to course teams.
+    """
+    return ContentErrorData(
+        message=(
+            f'<chapter> with url_name="{section.location.block_id}" and '
+            f'display_name="{section.display_name}" contains a '
+            f'<{duplicate_child.location.block_type}> tag with '
+            f'url_name="{duplicate_child.location.block_id}" and '
+            f'display_name="{getattr(duplicate_child, "display_name", "")}" '
+            f'that is defined in another section. Expected a unique '
+            f'<sequential> tag instead.'
+        ),
+        usage_key=_remove_version_info(duplicate_child.location),
+    )
```

Comment:
You might want to check with @mondiaz, but it's probably useful to indicate which instance of this sequence we did include in the course outline (i.e. which section has the first instance).